### ISSUE TITLE
P&T suspect something about Navigation is breaking the ability to make new Sections in Multisite...

### ISSUE THREAD

**Issue by MattWilcox** [verbb/navigation#267]
Tough one to describe, but Brad over at Pixel and Tonic has been [looking into an issue for me](https://github.com/craftcms/cms/issues/9121), and believes there's something up with Navigation that's having a knock-on effect, and making it impossible to create Sections that are enabled for more than one Site...

```
...I'm like 80% sure it's a bug/issue in the Verbb Navigation plugin with Singles and multisite Craft installs. Which I suppose I should have picked up earlier from the stack trace you shared in https://github.com/craftcms/cms/issues/9121

If you disable that plugin, you're able to save the "homepage" Single fine.
```

Is this something you've heard of? More details in the linked Issue, and I can provide composer.json/lock and a test database dump that has the issue.

**Comment by MattWilcox** [comment]
Update; we have a Navigation that we only want enabled on one Site. It's set accordingly...

<img width="580" alt="Screenshot 2021-07-08 at 09 00 51" src="https://user-images.githubusercontent.com/814039/124885354-16466780-dfcb-11eb-849a-e84a952d1d01.png">

But if we enable that Navigation for all Sites, then the original issue about being unable to create new Sections is "fixed".

**Comment by engram-design** [comment]
Yep, this indeed looks like a Navigation issue. I'll investigate and report back!

Can you also confirm the version of Navigation you're on?

If you can send everything to `web at verbb.io` that'll help!

**Comment by engram-design** [comment]
Can you also confirm the version of Navigation you're on?

**Comment by MattWilcox** [comment]
Let me know if those didn't get to you!

**Comment by engram-design** [comment]
Yep, received, thanks! Sorry for the delay.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-3 as 1.4.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.20](https://github.com/verbb/navigation/releases/tag/1.4.20)

**Comment by MattWilcox** [comment]
Cheers!
